.travis.yml

```diff
@@ -1,24 +1,25 @@
 language: php
 
-sudo: false
-
 cache:
   directories:
     - $HOME/.composer/cache/files
 
-php:
-  - 7.4
-
 before_script:
   - composer install
   - git config --global user.email "ocramius+travis-ci@gmail.com"
   - git config --global user.name "Marco Pivetta (via Travis-CI)"
 
-script:
-  - vendor/bin/phpcs
-  - vendor/bin/psalm
-  - vendor/bin/phpunit
-  - vendor/bin/infection --min-msi=99 --min-covered-msi=100
-  - php build-conflicts.php
-  - vendor/bin/roave-no-leaks
-  - mkdir -p /tmp/dependency-check && composer require --dev -d/tmp/dependency-check maglnet/composer-require-checker:^2.1.0 && /tmp/dependency-check/vendor/bin/composer-require-checker check composer.json
+jobs:
+  include:
+    - php: 7.4
+      script:
+        - vendor/bin/phpcs
+        - vendor/bin/psalm
+        - vendor/bin/phpunit
+        - vendor/bin/infection --min-msi=99 --min-covered-msi=100
+        - php build-conflicts.php
+        - vendor/bin/roave-no-leaks
+        - mkdir -p /tmp/dependency-check && composer require --dev -d/tmp/dependency-check maglnet/composer-require-checker:^2.1.0 && /tmp/dependency-check/vendor/bin/composer-require-checker check composer.json
+    - php: 7.4
+      if: branch = master
```

Comment:
This doesn't seem to be picked up. WTF Travis? :-\

Reply:
@slash3b do you have any idea of what might be happening here?

Reply:
honestly dunno, but let me dig into this. I'm going to try and tomorrow's evening :muscle:

Reply:
`php build-conflicts.php` mentioned in both stages :relaxed:    

I went all the way and created a test repo and added travis to it :rofl:    
Let me know if you want me to create a PR to this branch. Have a great rest of the evening ^^

Reply:
True that, but the build was still running this stage 🤔

Reply:
please take a look at the result of this [pipeline](https://travis-ci.org/Roave/SecurityAdvisoriesBuilder/builds/657183892?utm_medium=notification&utm_source=github_status) 
I'm wondering if the PR goes to master, does Travis run pipeline for both branches — master and the other one?
In my case above only one pipeline was run. 

I'll investigate further...

Reply:
Hmm, yeah, here it did run for `master` :|

I wonder if I should add an explicit conditional

Reply:
also found in [documentation](https://docs.travis-ci.com/user/pull-requests/#double-builds-on-pull-requests) — "If you see two build status icons on your GitHub pull request, it means there is one build for the branch, and one build for the pull request itself"

Reply:
do we have Travis configured like here — https://medium.com/@pck/travis-ci-avoiding-duplicate-builds-in-pull-requests-prs-but-still-build-deploy-updates-to-9bffa8ce3cdd ?

Reply:
Yes, those are the defaults AFAIK

Reply:
Oh wait, indeed! It did simulate a merge... Not sure how we can prevent that :| Maybe just `if [...] ` in bash then?

Reply:
If "master" build should be prevented in PR then it could be done with:
```yml
branches:
  except:
    - master
```
But then how to run build for master branch ?
Should we prevent that? :thinking: 


As a suggestion I've created a tiny PR for this branch — [PR](https://github.com/Roave/SecurityAdvisoriesBuilder/pull/119) but you won't see there how it would look for PR to master.
You may check how PR looks for master here — [PR in my fork as example](https://github.com/slash3b/SecurityAdvisoriesBuilder/pull/2)